blaze-core/src/main/scala/org/http4s/blazecore/Http1Stage.scala

```diff
@@ -75,7 +75,7 @@ trait Http1Stage[F[_]] { self: TailStage[ByteBuffer] =>
       rr: StringWriter,
       minor: Int,
       closeOnFinish: Boolean): Http1Writer[F] = lengthHeader match {
-    case Some(h) if bodyEncoding.map(!_.hasChunked).getOrElse(true) || minor == 0 =>
+    case Some(h) if bodyEncoding.forall(!_.hasChunked) || minor == 0 =>
```

Comment:
Good use of `forall`!

Reply:
That was an autofix suggestion by IntelliJ :)